src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -456,6 +502,28 @@ void ImageReplayer<I>::handle_init_remote_journaler(int r) {
     return;
   }
 
+  {
```

Comment:
But this is the init method for the remote journal, not the local journal.  The local journal is opened when the local image is opened and the exclusive lock is acquired (in ```OpenLocalImageRequest``` invoked from ```BootstrapRequest```).